vbfml/scripts/err_analysis

```diff
@@ -512,16 +529,40 @@ def roc(ctx, input_dir: str) -> None:
     features, predictions, label_encoding, labels = load_cache(input_dir)
     mjj = features["mjj"]
     detajj = features["detajj"]
+    weights = features["weight"]
 
     # compute ROC and precision recall curve
-    precision, recall, thresholds_prc = precision_recall_curve(labels, predictions)
-    fpr, tpr, thresholds_roc = roc_curve(labels, predictions)
+    fpr, tpr, thresholds_roc = roc_curve(labels, predictions, sample_weight=weights)
+    precision, recall, thresholds_prc = precision_recall_curve(
+        labels, predictions, sample_weight=weights
+    )
+
+    # add a dummy extra value above 1 to have a list of the same length
+    thresholds_prc = np.append(thresholds_prc, 2)
```

Comment:
Just to make sure I get it: Is it necessary for the later sorting function?

Reply:
yes exactly because it needs to be the same length as all the other variable (precision and recall here)

Reply:
Great, thanks for clarifying.